.travis.yml

```diff
@@ -19,9 +19,8 @@ cache:
 
 python:
     - 2.7
-    - 3.5
     - 3.6
-      # not yet - 3.7
+    - 3.7
```

Comment:
this won't work. Set the language e.g. to 'c' or 'generic' and use `PYTHON_VERSION` to control the python version.

Reply:
(since everything is set up from scratch in a conda env it doesn't really matter anyway what VM you start the tests with)